core/input/input.cpp

```diff
@@ -568,6 +568,22 @@ void Input::_parse_input_event_impl(const Ref<InputEvent> &p_event, bool p_is_em
 				}
 
 				_parse_input_event_impl(button_event, true);
+
+				// Reseting mouse position on release
+				// if we are emulating mouse from touch
```

Comment:
```suggestion
				// if we are emulating mouse from touch.
```
Code style requires that comments end with a period